packages/cli/src/lib/builder/buildTypeDefinitionsWorker.ts

```diff
@@ -33,12 +36,19 @@ export async function buildTypeDefinitionsWorker(
     );
   }
 
-  const { dirname } = require('path');
+  const { dirname, join } = require('path');
   const { entryPoints, workerConfigs, typescriptCompilerFolder } = workerData;
 
+  const { readFileSync, writeFileSync } = require('fs');
+
   const apiExtractor = require('@microsoft/api-extractor');
   const { Extractor, ExtractorConfig, CompilerState } = apiExtractor;
 
+  const ts: tsNamespace = require('typescript');
```

Comment:
```suggestion
  const ts: typeof import('typescript') = require('typescript');
```

Another way to do that which gets rid of some of the bonus code :grin: